android/app/src/main/java/com/google/codeu/chatme/view/adapter/UserListAdapter.java

```diff
@@ -97,5 +110,28 @@ private void setHolderPicture(String picUrl) {
                         .into(this.civUserPic);
             }
         }
+
+        @Override
+        public void onClick(View view) {
+            // Create new conversation object, set Owner and Id, and add participant
+            Conversation conversation = new Conversation(createConversationPresenter.getUserId());
+            conversation.setId(createConversationPresenter.newConversationId());
```

Comment:
This suggests that our database structure would look as follows:
```
+--- {conversationId}
     +--- {conversationId}
     +--- {authorId}
     +--- {participants}
```
We probably don't need the `conversationId` as a field in our database.